mathics/doc/common_doc.py

```diff
@@ -1329,8 +1322,12 @@ def get_tests(self):
         # A guide section's subsection are Sections without the Guide.
         # it is *their* subsections where we generally find tests.
         for section in self.subsections:
+            if not section.installed:
+                continue
             for subsection in section.subsections:
                 # FIXME we are omitting the section title here...
+                if not subsection.installed:
+                    continue
```

Comment:
I am also not seeing this continue or the one on 1326 getting used either.

Reply:
OK, thecheck is done in *doctest.get_tests*, so even if we reach there, there are not tests yield.

Reply:
Ok. I guess we should delete this then. @mmatera would you finish this off along with the IS_PYPY stuff. I'd like to be done looking at this. Thanks.

Reply:
And maybe you could also remove the homegrown caching stuff.

Homegrown caching seems to be a Mathics specialty.

Reply:
Yep. It does not change too much anything, since internally, Python does the same.

Reply:
What I would keep is the ``check_requires_list``, because avoids to repeat code.

Reply:
Done.